packages/block-library/src/navigation/index.php

```diff
@@ -201,4 +233,4 @@ function block_core_navigation_typographic_presets_backcompatibility( $parsed_bl
 	}
 	return $parsed_block;
 }
-add_filter( 'render_block_data', 'block_core_navigation_typographic_presets_backcompatibility' );
+	add_filter( 'render_block_data', 'block_core_navigation_typographic_presets_backcompatibility' );
```

Comment:
This file will be copied over to WP core so it should be properly formatted according to coding guidelines:
```suggestion
add_filter( 'render_block_data', 'block_core_navigation_typographic_presets_backcompatibility' );
```

Reply:
Thanks for pointing it out, for some reason `phpcs` does not auto format this even though it's being called out by the linter.